translations/en.json

```diff
@@ -538,7 +538,9 @@
 				"date-time": "Date/Time",
 				"from": "From",
 				"to": "To",
-				"usd-value": "USD Value"
+				"usd-value": "USD Value",
+				"no-trade-history": "You have no trades yet",
```

Comment:
Can you add a dot to the end of this sentence, so it's in line with the other one? :)